packages/presentation/lib/core/theme/style_text.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/core/theme/theme_app.dart';
+
+const TextStyle headline1 = TextStyle(
+    fontSize: 22,
+    fontWeight: FontWeight.w600,
+    fontFamily: "Volte Rounded",
+    color: AppColors.textMain);
+const TextStyle headline2 = TextStyle(
+  color: AppColors.textMain,
+  fontSize: 16,
```

Comment:
I think best way to use static numbers and other params is constants file

Reply:
why? static is best when you write them in class